docker/code-formatting-tools.dockerfile

```diff
@@ -9,7 +9,7 @@
 #   docker build \
 #       --tag=newsboat-code-formatting-tools \
 #       --file=docker/code-formatting-tools.dockerfile \
-#       docker
+#       .
```

Comment:
Why is this needed? It doesn't look like Dockerfiles use anything outside the docker directory.

Reply:
This is not strictly necessary, however, I prefer to keep these consistent between all of our docker files.
In the buildtools docker files, I've used a `COPY` command to pull in the new `entrypoint.sh` so there it is was necessary to update the path (cirrus seems to always specify the root directory of the git checkout, without option to override it so I did have to include the `docker/` part in the `COPY` command)